rpm/lipstick-qt5.spec

```diff
@@ -28,6 +28,7 @@ BuildRequires:  pkgconfig(contentaction5)
 BuildRequires:  pkgconfig(mlite5) >= 0.2.19
 BuildRequires:  pkgconfig(mce) >= 1.22.0
 BuildRequires:  pkgconfig(mce-qt5) >= 1.4.0
+#                                     ^^^^^ TBD/1.5.0
```

Comment:
TBD

Reply:
Now that libmce-qt has been tagged: 1.5.0 it is